BFF/v3/docs/content/_index.md

```diff
@@ -0,0 +1,14 @@
++++
+title = "BFF Security Framework"
+date = 2020-09-10T08:20:20+02:00
+weight = 95
+chapter = true
++++
+
+# BFF Security Framework
+
+The Duende.BFF (Backend for Frontend) security framework packages the necessary components to secure browser-based frontends (e.g. SPAs or Blazor applications) with ASP.NET Core backends.
+
+Duende.BFF is free for development, testing and personal projects, but production use requires a [license](https://duendesoftware.com/products/bff#pricing). [Special offers](https://duendesoftware.com/specialoffers) may apply.
+
+The source code for the BFF framework can be found [here](https://github.com/DuendeSoftware/products/tree/main/bff). Builds are distributed through NuGet [here](https://www.nuget.org/packages/Duende.BFF/), and samples are available [here]({{< ref "/samples/bff" >}}).
```

Comment:
So the use of *here* can be problematic for SEO and crawlers. "Here" doesn't give crawlers enough context. Try linking the the phrase that describes the link. For example, "the source code for the BFF framework".